galley/pkg/runtime/flow/table.go

```diff
@@ -0,0 +1,107 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package flow
```

Comment:
I really suggest renaming this. Flow & table immediately conjure up concepts from the networking world (a flow is a connection between two endpoints, a 5 tuple ..).. Atleast call this dataflow instead of just flow.

Reply:
renamed.